completion/ksbaduk.cwl

```diff
@@ -0,0 +1,103 @@
+# ksbaduk package
+# ponte-vecchio 2023-01-17 for v0.6.4
+
+#keyvals:\usepackage/ksbaduk#c
+ball
+plain
+badukpancolor=#%color
+badukpansize=%<number%>
+posmark
+imageback
+imagefile=%<file%>
+textcmds
+numberfont=%<font%>
+tmarkfont=%<font%>
+#endkeyvals
+
+## 2.2 Global options
+\BadukpanSize{%<number%>}%definition
```

Comment:
I doubt that %definition is doing anything here.
Not sure what the intention is ?

Reply:
Removed in the new commit